cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -245,6 +250,23 @@ func generateMountsCommonInstallRecover(recoverySystem string) (allMounted bool,
 		return false, nil
 	}
 
+	// 1a. measure model
+	err = stampedAction(fmt.Sprintf("%s-model-measure", recoverySystem), func() error {
```

Comment:
this should be "*-measured"  but seems there's no test that would double check for this ATM